app/templates/userdashboard.html

```diff
@@ -0,0 +1,90 @@
+<!DOCTYPE html>
+{% extends "base.html" %}
+
+{% block head %}
+{{ super() }}
+
+{% endblock %}
+
+<body>
+{% block page_content %}
+
+<form method="post">
+    <div class="items-container">
+        <button type="submit" class="new-btn"><a href="{{ url_for('create_shopping_lst') }}">Create A ShoppingList</a>
+        </button>
+        <div class="form-header">
+            <h2 class="title">Awesome ShoppingList App</h2>
+        </div>
+
+        <div class="card">
+            {% if current_user.get_num_of_shopping_lists() > 0 %}
+            {% for shoppinglist_name, shopping_list in current_user.shopping_lists.items() %}
+
+            <form method="post">
+                <div class="form-header">
+                    <h4 class="title"> {{ shoppinglist_name }} </h4>
+                </div>
+                <div class="form-body">
+                    <ol>
+                        {% if shopping_list.get_categories() > 0 %}
+                        {% for key, cat in shopping_list.categories.items() %}
+                        <ul>
+
+                            <strong>{{ key }}</strong>
+                            {% for item_name, item in cat.items() %}
+                            <li>{{ item.quantity }} {{ item.name }}(s)
+                                Costing Sh {{ item.price }} Each
+                                <button style="height:20px;width:50px;margin-bottom:10px" type="button"
```

Comment:
Should avoid in-line styles as much as possible. Extract these to a class and migrate the styling to the css files.